src/components/score.js

```diff
@@ -0,0 +1,10 @@
+import { updateScoreElement } from '../views/scoreView.js';
+
+export let currentScore = 0;
+
+export const updateScoreValue = (currentQuestion, userAnswer) => {
+  if (currentQuestion.correct === userAnswer) {
+    currentScore++;
+    updateScoreElement(currentScore);
+  }
+};
```

Comment:
Separating out the logic for updating the score into a separate file veers off a bit from the original structure of the project which is to keep the markup files in the views folder and the user interaction functionality in the pages folder. 
If you think the score value will need to be updated also from other pages than from the `questionPage` then I'm fine with this approach. If the `questionPage` will be the only place where the app needs to react to the user interaction by updating the score value then I'd suggest moving the function inside the `questionPage` file to stay consistent with the existing app structure.

Reply:
That's a fair point, I agree with you. I moved "updateScoreValue" function to the questionPage file.